testing/kfctl/kfctl_go_test.py

```diff
@@ -78,6 +78,9 @@ def test_build_kfctl_go(app_path, project, use_basic_auth, use_istio):
   else:
     init_args.append("--use_istio=false")
 
+  if package_manager:
+    init_args.append("--package-manager kustomize")
```

Comment:
If this is a string why are you taking a boolean as an argument to the test? Why don't you just pass through the value directly?

Reply:
Agreed. I thought since we had just the two to choose from (ksonnet and kustomize) I'd use a boolean